### DISCUSSION TITLE
Paper reference for using MONAI

### DISCUSSION THREAD

**Discussion by ghost** [Project-MONAI/MONAI#1519] [General]
Hi,
I'm helping write a conference abstract and wondered if there is any publication you'd suggest referencing for MONAI? Of course I can find them for the underlying architectures etc but I've been unable to find one for the MONAI platform, is there one yet?
Thanks,
Jeff.

**Comment by wyli** [comment]
please see https://github.com/Project-MONAI/MONAI/issues/1166

**Reply by ghost** [reply]
Thank you, apologies for not spotting this! I do suggest adding it to the website
